content/zh/news.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 社区快讯
+sidebar: false
+---
+
+### 2021 NumPy survey
```

Comment:
The news page has been override by @InessaPawson 's ["Make an announcement about the 2021 NumPy survey"](https://github.com/numpy/numpy.org/pull/453),  Not only the Chinese translation is affected, but the translation of other languages is also affected. should we fix this content? It's looks like the translation process is conflicted with document maintenance.

Reply:
Ah, I think this is what I mentioned at #55